client/src/components/App.js

```diff
@@ -32,6 +35,12 @@ function App() {
         <Route path="/cart">
           <Cart />
         </Route>
+        <Route path="/confirmation">
```

Comment:
Hi @n4dz, 
Can we make the redirect point to 'order-confirmation' instead
I would also adjust the other one to be consistent.

Reply:
Hi @carldumas,
I made the changes 👍